webalyzer/collector/urls.py

```diff
@@ -5,5 +5,10 @@
 
 urlpatterns = patterns(
     '',
+    url(
+        r'^check/(?P<source_type>[\w]+)/(?P<source_hash>[\-\w\.]+)$',
```

Comment:
Again, this ought to use the domain too. 
Not only is the hash quite small (in combinations), but if we have any requests against the server by domain you'd be able to get a better control over the activity per domain.